templates/blocks/share_buttons.twig

```diff
@@ -37,6 +37,16 @@
             <span class="visually-hidden">{{ __( 'Share via', 'planet4-master-theme' ) }} Email</span>
         </a>
     {% endif %}
+    <!-- Bluesky -->
+    {% if share_platforms.bluesky %}
+        <a href="https://bsky.app/intent/compose?text={{ social.title|url_encode }}{% if share_text or social.description %} - {{ (share_text ?? social.description)|striptags|url_encode }}{% endif %}, via @{{ social_accounts.bluesky }} {{ (socialLink ~ '&utm_source=bluesky')|url_encode }}"
+           onclick="window.dataLayerPush('Bluesky');"
```

Comment:
If there is no bluesky account set up in footer social accounts, we get something like this: `via @ <url>`. Which doesn't look nice. I think we have the same issue for X.